src/utils/health-check.ts

```diff
@@ -70,15 +90,23 @@ const warnIfFreeBondIsLessThanMinDiggingFee = (
       `\n   Your free bond is less than your minimum digging fee. You will not be able to accept new jobs!`
     );
     archLogger.error(
-      `   Run: \`npm run start -- --deposit-bond:<amount>\` to deposit some SARCO\n`
+      `   Run: \`cli update -f <amount>\` to deposit some SARCO\n`
     );
   }
 };
 
 const warnIfEthBalanceIsLow = (ethBalance: BigNumber): void => {
   if (ethBalance.lte(ethers.utils.parseEther("0.0005"))) {
+    archLogger.error(
+      `\nYou have very little ETH in your account: ${ethers.utils.formatEther(ethBalance)} ETH.\nYou may not be able to sign any transactions (or do unwrappings)!\n`
+    );
+  }
+};
+
+const warnIfSarcoBalanceIsLow = (sarcoBalance: BigNumber): void => {
+  if (sarcoBalance.lte(ethers.utils.parseEther("0.0005"))) {
     archLogger.warn(
-      `\n   You have very little ETH in your account. You may not be able to sign any transactions (or do unwrappings)!\n`
+      `\nYou have very little SARCO in your account: ${ethers.utils.formatEther(sarcoBalance)} SARCO\n`
```

Comment:
Having an alert for low eth balances could be really useful because publishing will fail if the archaeologist doesn't have enough ether. Not something to handle in this PR but a thought for the future

Reply:
@katien  -- the function check still exists above -- I just broke this into its own SARCO method -- the service will still currently warn you if you have have low/zero ETH balance.